.github/workflows/docker-sched.yml

```diff
@@ -1,4 +1,4 @@
-name: [Scheduled] Build and upload docker images
+name: "[Scheduled] Build and upload docker images"
```

Comment:
Is this only needed because of the `[]`? There are a lot of other `name` which are not quoted.